docs/docs/components.mdx

```diff
@@ -2551,7 +2551,7 @@ Selectors predict a bot response from a set of candidate responses.
               "id": 1388783286124361986,
               "confidence": 0.7,
               "intent_response_key": "chitchat/ask_weather",
-              "response_templates": [
+              "responses": [
```

Comment:
@wochinge Correct me if I'm wrong but I don't think we can change the output schema of the component in a minor version?

Reply:
Primarily because this will break the assistants which rely on the output for some custom action code or any other code outside of rasa, basically equivalent to an API change.

Reply:
True. Good point. Wasn't thinking of external users accessing that information. @yennycheung I suggest to rename the key back to it's old thing then.

Reply:
Let me give this a try.